### ISSUE TITLE
Package installation throws errors

### ISSUE THREAD

**Issue by n-gao** [TUM-DAML/gemnet_pytorch#9]
There were several issues prohibiting the installation of gemnet as package:
* The `setup.py` file referred to a `gemnet_pytorch` package even though the folder name is `gemnet`
* `python`, `cudatoolkit`, `pytorch` and `torch_geometric` are invalid `pip` packages.

Most of these should be fixed by 1922774fbf8e21179de5c64d7490fe8dc28fa720

**Comment by gasteigerjo** [comment]
Thank you!

Rather than deleting the pytorch and torch_geometric dependencies we should specify the correct pip names. Especially the version number can be important.

Similarly for the Python version, which should say something like [`python_version>=3.8`](https://peps.python.org/pep-0508/#environment-markers).

**Comment by n-gao** [comment]
Good point! If one wants to install `torch` and `torch_geometric` via `pip install -r requirements.txt` the CUDA version must be fixed which may not be desired. 
For instance, for  CUDA 10.2 one could add
```
torch==1.10
```
However, for CUDA 11.3 it must be
```
--extra-index-url https://download.pytorch.org/whl/cu113
torch==1.10
```
For `torch_geometric` it gets more complicated with the CUDA version since you also need the `torch` version.

I am not sure whether PEP508 is the correct solution regarding the python version. It more or less makes the dependence of a package depending on the python version but if you want to fix the python version you should require a minimum and maximum in your `setup.py`, see [`python_requires`](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#python-requires).

**Comment by gasteigerjo** [comment]
Fixed in 0110dce636e2ded96e744043f7c6d5d3570e7871.

I don't think we need to specify CUDA at all. Imho, the user is responsible for this if they want to use CUDA.
